### PR TITLE
Add per-bucket config to catalog

### PR DESCRIPTION
**Pull request by meffij**
Prototype catalog config with per-bucket configs.

TODOs:

- [ ] update Quilt icon

- [ ] update Quilt description

- [ ] update AICS icon

- [ ] update AICS description

### CODE REVIEW

**Comment by meffij** [comment]
I think more information than just an endpoint URL will be necessary to make successful signed search requests. We will need at least the AWS region of the search endpoint. @nl0 Is there anything else we need to make search work correctly?

**Comment by nl0** [comment]
@meffij i *think* it's enough (it was enough for alpha endpoint). i'll let you know if i stumble upon any problems getting search operational

**Comment by nl0** [comment]
@meffij implemented in #125

File: catalog/static/config.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "region":"us-east-1",
+    "s3Bucket":"t4-bucket",
+    "apiGatewayUrl":"https://tdfqht5b70.execute-api.us-east-1.amazonaws.com/prod",
+    "elasticSearchUrl":"https://search-t4-stack-vh6cadfyzemgwbdmpfjqenq36a.us-east-1.es.amazonaws.com"
+  },
+  "configs":{
+    "quilt-example":{
+      "icon":"https://quiltcorp.files.wordpress.com/2017/08/q.png",
+      "title":"Quilt Data",
+      "description":"Turbocharging your S3 Bucket",
```
Comment (akarve) [inline catalog/static/config.json]: T4 transforms an S3 bucket into a visual data regsitry

File: catalog/static/config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "api":"not used but needs to be set :(",
+  "aws":{
+    "region":"us-east-1",
+    "s3Bucket":"t4-bucket",
+    "apiGatewayUrl":"https://tdfqht5b70.execute-api.us-east-1.amazonaws.com/prod",
+    "elasticSearchUrl":"https://search-t4-stack-vh6cadfyzemgwbdmpfjqenq36a.us-east-1.es.amazonaws.com"
+  },
+  "configs":{
+    "quilt-example":{
+      "icon":"https://quiltcorp.files.wordpress.com/2017/08/q.png",
```
Comment (akarve) [inline catalog/static/config.json]: https://d1zvn9rasera71.cloudfront.net/q-128-square.png

File: catalog/static/config.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      "title":"Quilt Data",
+      "description":"Turbocharging your S3 Bucket",
+      "search_endpoint":"https://search-quilt-example-rytakqayno3rlembetdhuflrna.us-east-1.es.amazonaws.com/"
+  },
+  "quilt-aics": {
+      "icon":"https://3c1703fe8d.site.internapcdn.net/newman/gfx/news/hires/2017/3dwindowinto.png",
+      "title":"Allen Institute for Cell Science",
+      "description":"Launched with a contribution from Paul G. Allen in 2014, the Allen Institute for Cell Science will serve as a catalyzing force to integrate diverse technologies and approaches at a large scale in order to study the cell as an integrated system: something that traditional academic labs cannot do. Our inaugural project, the Allen Cell Explorer, will be a novel visual, dynamic, predictive model of the cell that will accelerate cell biology and biomedical research.",
```
Comment (akarve) [inline catalog/static/config.json]: "The Allen Cell Explorer is a novel visual, dynamic, predictive model of the cell that will accelerate cell biology and biomedical research."

**Pull request closed without merging**